meta-lmp-bsp/conf/machine/apalis-imx8.conf

```diff
@@ -55,7 +55,8 @@ IMX_DEFAULT_BOOTLOADER = "u-boot-fio"
 
 MACHINE_FIRMWARE:append = " linux-firmware-pcie8997"
 MACHINE_FIRMWARE:append = " firmware-imx-vpu-imx8"
-MACHINE_FIRMWARE:append = " firmware-imx-8"
```

Comment:
I'm a bit confused as firmware-imx-8 seems to be dropped only on meta-freescale master, and we are still working with meta-freescale kirkstone, right?

Reply:
> I'm a bit confused as firmware-imx-8 seems to be dropped only on meta-freescale master, and we are still working with meta-freescale kirkstone, right?

meta-freescale kirkstone doesn't include support for NXP BSP LF6.1, so we should switch to master for that. Otavio suggested doing this.

Reply:
But is this on preparation for switching to master?
I think we would still use a lmp linux recipe, so it's not dependent on the meta-freescale being on master or kirkstone

Reply:
> But is this on preparation for switching to master? I think we would still use a lmp linux recipe, so it's not dependent on the meta-freescale being on master or kirkstone
> 
> in case this is already for the preparation to switch to master, we also need: [angolini@174080a](https://github.com/angolini/meta-lmp/commit/174080a0bbe11c5c7ece14606a645bbc9d33676a)
> 
> But I would prefer to keep the "switching to master" PR separated from the "updating kernel to 6.1" PR

Yes, we have to switch to master. It is still compatible with kirkstone and supports NXP BSP LF6.1